### PR TITLE
Fixed stuff that needed to be fixed

### PR DESCRIPTION
**Pull request by snemes**
This PR fixes a few things that are actually needed to make this work. :)

Update: Apologies, I might have exaggerated a bit, not every change here is required to make this work. ;)

### CODE REVIEW

File: Deep-Drop/core/deepdrop.py [hunk 1]
```diff
@@ -33,30 +32,26 @@ def parse_process_list(process_list):
 
     processes = namedtuple('process', 'pid, owner, name')
 
+    hostname = ''
```
Comment (moohax) [inline Deep-Drop/core/deepdrop.py]: Good catch. This code came from parsing lots of process lists. 

will return False on exception.

File: Deep-Drop/core/config.py [hunk 2]
```diff
@@ -8,17 +8,18 @@
...
-    'vba': os.path.join(basedir, 'macros\\basic.vba')
+    'vba': os.path.join(basedir, 'macros', 'template.vba'),
+    'vbs': os.path.join(basedir, 'macros', 'template.vbs')
 }
 
 # data
 data_files = {
-    'decisiontree': os.path.join(basedir, 'data\\data_process_count.csv'),
-    'neuralnetwork': os.path.join(basedir, 'data\\data_process_count.csv')
+    'decisiontree': os.path.join(basedir, 'data', 'data_process_count.csv'),
```
Comment (moohax) [inline Deep-Drop/core/config.py]: Keep.

File: Deep-Drop/core/deepdrop.py [hunk 3]
```diff
@@ -33,30 +32,26 @@ def parse_process_list(process_list):
...
         return 'Hello'
-        
+
     for process in task_result[1:]:
         full_result = re.findall(r'^([0-9]+)\s+(\S+)\s+(.+)$', process)
 
         if full_result:
-            (pid, owner, name) = full_result[0]
-            
+            pid, owner, name = full_result[0]
```
Comment (moohax) [inline Deep-Drop/core/deepdrop.py]: Not functionally different.

File: Deep-Drop/core/macros/template.vbs [hunk 4]
```diff
@@ -1,58 +1,53 @@
-Sub OnOpen()
```
Comment (moohax) [inline Deep-Drop/core/macros/template.vbs]: vbs/vba/txt just needs copy + pasting into a doc.

File: Deep-Drop/core/deepdrop.py (hunk 3 above)
Comment (snemes) [reply Deep-Drop/core/deepdrop.py]: Indeed. It's just a style fix to remove the unnecessary parentheses. If your personal preference is to use parentheses around tuple assignments, that's fine too.

File: Deep-Drop/core/macros/template.vbs (hunk 4 above)
Comment (snemes) [reply Deep-Drop/core/macros/template.vbs]: For some reason Git shows this as if I renamed and modified template.vbs, but that's not the case. The template.vba file is a copy of template.vbs with some modifications to allow executing it via wscript.exe/cscript.exe for those poor souls who would like to test this on a VM without Microsoft Word installed (like me).

Comment (moohax) [reply Deep-Drop/core/macros/template.vbs]: That's fair, this is obviously in the context of a phishing email - but using this internally to detect a safe pivot or something could be an interesting use case.

**Comment by moohax** [comment]
Thanks for the pull request. Really appreciate it the input. Let me know if you have success with it.

**Pull request closed without merging**